tests/Feature/SongTest.php

```
@@ -28,6 +29,25 @@ public function index(): void
         $this->getAs('api/songs?sort=title&order=desc')->assertJsonStructure(SongResource::PAGINATION_JSON_STRUCTURE);
     }
 
+    #[Test]
+    public function indexSortedByFavoriteScopesToCurrentUser(): void
+    {
+        $user = create_user();
+        $other = create_user();
+
+        $unfavorited = Song::factory()->createOne(['title' => 'Unfavorited']);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$unfavorited'. (undefined)

(UnusedLocalVariable)